sections/csgcn/csgcn-chapter.tex

```diff
@@ -291,8 +291,11 @@ \subsection{Training}\label{subsec:csgcn_is_training}
 $\hat{y}$ is a score prediction for respectively a positive interaction $(u,i,c)$ between a user $u$ and an item $i$ in context $c$, and a negative interaction $(u,j,c)$ for the same user in the same context, but with a sampled non-interaction $j$.
 \\
 The positive interaction is an observed interaction between a user and an item, which can be found in the training set.
-We then randomly select an item that the same user has not interacted with in this given context as a negative sample, with the assumption that an unobserved interaction is equal to a negative interaction when working with implicit data.\\
-While using sampling for calculating BPR is biased \cite{nonsampling}, it is usually necessary since using non-sampling not only requires the method to look at each possible interaction from the dataset, but also to consider each non-observed interaction as a negative data.
+We then randomly select an item that the same user has not interacted with in this given context as a negative sample, with the assumption that an unobserved interaction is equal to a negative interaction when working with implicit data.\\ 
+Using negative sampling for calculating loss is biased \cite{nonsampling}.
```

Comment:
måske udvidde med at det er biased fordi at performance afhænger meget af om det er et heldigt eller uheldig sample man lige får fat i

Reply:
there it is